src/ExponentialMap.jl

```diff
@@ -253,15 +253,15 @@ This follows from ``\\exp(-M)⋅\\exp(M) = I`` for any ``M``.
 julia> em = ExponentialMap(SparseMatrixExp(SparseMatrixCSC([2.0 0.0; 0.0 1.0])),
                            BallInf([1., 1.], 1.));
 
-julia> ∈([5.0, 1.0], em)
+julia> ∈([-1.0, 1.0], em)
```

Comment:
usually we need a space before each `julia> ` for the test to work. does it mean that we are not doctesting 

```julia
∈(x::AbstractVector{N}, em::ExponentialMap{<:LazySet{N}})::Bool where {N<:Real}
```

Reply:
No, the lines below each command must be equal to the output in the REPL. If the command ends with a `;`, the REPL just prints an empy line. Hence you need an empty line *after* each `julia>` that ends in `;`. The doctests worked here.